File: car-listing-service/src/main/java/io/classiccrew/car_listing_service/entity/CarDetails.java
package io.classiccrew.car_listing_service.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@Getter
@Setter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class CarDetails extends BaseEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO, generator = "native")
    private Long carDetailId;
    private Long carId;
    private String mileage;
    private String color;
    private String vin;
    private String status;
}

File: car-listing-service/rental/core/entities/car_details.py
from datetime import datetime
from entities.base_entity import BaseEntity


class CarDetails(BaseEntity):
    def __init__(self,
                 car_detail_id: int,
                 car_id: int,
                 mileage: str,
                 color: str,
                 vin: str,
                 status: str,
                 created_at=None, created_by=None, updated_at=None, updated_by=None):

        super().__init__(created_at, created_by, updated_at, updated_by)

        self.car_detail_id = car_detail_id
        self.car_id = car_id
        self.mileage = mileage
        self.color = color
        self.vin = vin
        self.status = status
